lib/Settings/UserFlags.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace MailPoet\Settings;
+
+use MailPoet\Models\UserFlag;
+use MailPoet\WP\Functions as WPFunctions;
+
+class UserFlags {
+
+  /**
+   * @var array|null
+   */
+  private static $data = null;
+
+  public function getDefaults() {
+    return [
+      'last_announcement_seen' => false,
+      'editor_tutorial_seen' => false,
+    ];
```

Comment:
I think we need more flags than these two. I just quickly found those:
- show_poll_success_delivery_preview
- show_congratulate_after_first_newsletter
- mailpoet_skip_welcome_wizard
- user_seen_editor_tutorial
- show_intro

Reply:
I am not sure about `mailpoet_skip_welcome_wizard` since the user can edit settings on the wizard. I think we should keep it per site. I will create a separate Jira task for adding other flags because this PR is already a large one.

Reply:
Ok, that is fair. Create a new JIRA. And you're right, the wizard is probably per site.